mala/network/multi_predictor.py

```diff
@@ -0,0 +1,342 @@
+"""Tester class for testing a network."""
```

Comment:
This file is largely a copy of `mala/network/predictor.py`, as such many docs/comments may or may not apply (e.g. "Tester class for testing a network." at the top, which is also in `predictor.py`).

The main diff is `load_run()` and some changes in `_forward_snap_descriptors()`. There is large code duplication which can become a maint issue later. I *don't think* a solution is to let `Predictor` support either a `network` or a list of them. A subclass is fine.

As far as I could diff things, there is *no diff* in `predict_from_qeout()` and `predict_for_atoms()` at all, so they can be removed from the subclass? Changes in `_forward_snap_descriptors()` are small, maybe one could put the common code in an extra method `_forward_snap_descriptors_core()` (bad name I know).

I know this is test code to get going quickly, therefore copy-hack-run is fine. Only once this is on `develop`, it is kind of a feature and someome has to maintain it :)

Reply:
Good point, I reduced the code duplication!